### DISCUSSION TITLE
Users graphql

### DISCUSSION THREAD

**Discussion by tonxxd** [directus/directus#5202] [Archive]
I created a collection called "users" that stores some contact details with an optional field "user" that links to the directus_users collection if an account is created. I tried to query it through the graphql endpoint

```graphql
users {
  name
  lastname
  user {
     role {
        name
     }
  }
}
```

The issue is that, since the graphql endpoint for directus_users is "users" as reported in the [docs](https://docs.directus.io/reference/api/system/users/#list-users) (I found out about it later) there is a conflict in the schema so I am not able to query the directus_users anymore. In addition, the user field in the graphql query shown above is not being populated with the data from the directus_users collection (its type is String in the schema even though the field type is O2M)

I think I will just solve it by renaming the collection something else but maybe it would be better to show a warning in the docs.

**Comment by rijkvanzanten** [comment]
The system collections are only available through `/graphql/system`, so this naming conflict doesn't exist 👍🏻

**Reply by tonxxd** [reply]
I understand, so I can't populate the O2M field "user" through graphql, right?

**Reply by rijkvanzanten** [reply]
You can, just use `/graphql/system` instead of `/graphql` to query/mutate `directus_users` on the root. All relationships exist regardless of whether you go "through" regular items or system items, so you can still use relations to system collections if you have relationships setup to them in your own collections
